core/editor/data/widgets-config/controller.php

```diff
@@ -0,0 +1,84 @@
+<?php
+namespace Elementor\Core\Editor\Data\WidgetsConfig;
+
+use Elementor\Core\Utils\Collection;
+use Elementor\Data\Base\Controller as Controller_Base;
+use Elementor\Plugin;
+
+class Controller extends Controller_Base {
+	/**
+	 * @return string
+	 */
```

Comment:
Here and next...
```suggestion
```
Useless, they should come from the base, the DOC should be set only if you override.